.gitignore

```diff
@@ -1,3 +1,4 @@
+*~
```

Comment:
?

Reply:
I'm a `emacs` user, I put this one everywhere.

Reply:
Ok, I prefer to only have language or build specific things the `.gitignore` file. If you have things that are specific to your local environment, please put them into your [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore).

Reply:
did not know of this option.  thank you, it's useful.